packages/shared/src/components/BookmarkEmptyScreen.tsx

```diff
@@ -1,12 +1,21 @@
 import React, { ReactElement } from 'react';
 import Link from 'next/link';
+import classNames from 'classnames';
 import { BookmarkIcon } from './icons';
 import { Button, ButtonSize, ButtonVariant } from './buttons/Button';
 import { EmptyScreenIcon } from './EmptyScreen';
+import { useMobileUxExperiment } from '../hooks/useMobileUxExperiment';
 
 export default function BookmarkEmptyScreen(): ReactElement {
+  const { isNewMobileLayout } = useMobileUxExperiment();
+
   return (
-    <main className="withNavBar inset-0 mx-auto mt-12 flex max-w-full flex-col items-center justify-center px-6 text-theme-label-secondary">
+    <main
+      className={classNames(
+        'withNavBar inset-0 mx-auto mt-12 flex max-w-full flex-col items-center justify-center px-6 text-theme-label-secondary',
+        isNewMobileLayout && 'tablet:pl-22',
```

Comment:
Shouldn't we solve this on the main wrapper somehow?
It feels kind of wrong doing it in every layout to be honest. (especially since they just center themselves already)

You can probably look at the current padding left on the main layout already.

Reply:
I think I initially modified this on [the main layout](https://github.com/dailydotdev/apps/blob/87ae3ef43a4dea208611cc17902ad0e087b0e603/packages/shared/src/components/MainLayout.tsx#L220-L233), but it turned out not to work everywhere as expected, but can't recall exactly why, let me double check

Reply:
Double checked and we can set this at the main layout level, indeed, updated 👍  thanks for noting this